DigitalLearningSolutions.Web/Views/Pricing/Index.cshtml

```diff
@@ -4,6 +4,9 @@
   var url = ConfigHelper.GetAppConfig()["CurrentSystemBaseUrl"] + "/pricing?nonav=true";
 }
 
-<div class="nhsuk-grid-row responsive-iframe-wrapper">
+<div class="nhsuk-grid-row">
+  <h1 class="nhsuk-heading-xl heading-margin-2" id="app-page-heading">Pricing</h1>
```

Comment:
Should the id be "page-heading" rather than "app-page-heading"?

Reply:
I just copied it from another one, but yeah probably. I'll change it